apps/tutorial-servo/main.cpp

```diff
@@ -0,0 +1,14 @@
+#include "CANMsg.h"
+#include "TutorialServo.h"
+CAN can(CAN_RX, CAN_TX);
+CANMsg rxMsg;
+CANMsg txMsg;
+
+int main() {
```

Comment:
This implimentation doesnt really use the CAN input, check out https://github.com/uwrobotics/MarsRover2020-firmware/blob/master/apps/test-can/main.cpp for an example of using CAN

Reply:
I gave that file a look and I changed up my driver file, however I don't fully understand how to process the CAN message into a command for my servo motor. Would it be possible for us to hop in another zoom call? I understand if you are busy with exams cuz things can get hectic 😂

Reply:
I'll probably be too busy this week but @younesr1 may be able to help

Reply:
I'll try to join the discord session after tomorrows meeting if i have time

Reply:
sounds good @younesr1 let me know if you have time to make a call

Reply:
Yeah for sure. We can chat about it before the meeting around 7 if youd like. if not we can chat at the work session.

Reply:
7 works for me perhaps you can message me on slack a zoom meeting around that time